pom.xml

```diff
@@ -147,6 +147,7 @@
     <module>service-base</module>
     <module>tests</module>
     <module>site</module>
+    <module>jmeter</module>
```

Comment:
I assume that this is also intended for e.g. "tests" between "service-base" and "site" .. so all should be on the lexically correct position. I will order all.